Dave-mash/StackOverflow-lite
# [finishes #162573275] Added sections for posting answers to questions

# What does this PR do?
Creates a text input area for posting an answer to a question
# Description of Task to be completed?
The tasks for this PR is:
* Add an input area for posting a question
# How should this be manually tested?
Clone this repository on your machine:
$ git clone https://github.com/Dave-mash/stackOverflow-lite.git

* CD into the project folder on your machine
$ cd stackOverflow-lite

* Check out the branch for this PR
$ git branch ft-user-login-162573275

* Open the index.html file on your browser

Pivotal Tracker story associated with this PR
Story-ID #162573275

Screenshot:
![post-answer](https://user-images.githubusercontent.com/41325679/49898282-a8b11980-fe69-11e8-8024-785fef303984.png)
